pkg/results/error.go

```diff
@@ -120,3 +120,20 @@ func DefaultReason(err error) error {
 
 	return ForReason(ReasonUnknown).ForError(err)
 }
+
+func ConvertToErrorList(errs []error) []*Error {
+	var ret []*Error
+	for _, e := range errs {
+		ret = append(ret, e.(*Error))
```

Comment:
Please do not put code that can just panic in non-test files

Reply:
addressed